mettle/src/process.h

```diff
@@ -65,4 +65,14 @@ int process_kill_by_pid(struct procmgr *mgr, pid_t pid);
  */
 pid_t process_get_pid(struct process *p);
 
```

Comment:
ideally, process knows nothing about channels and vice-versa. The channel id should be something we can derive from the callback arg (void *arg) instead. So, probably we need modify mettle/src/stdapi/sys/process.c to instead of storing a channelmgr  reference in arg, it should store a new struct that can also contain the channel ID opaquely from mettle/src/process.c's point of view.